sites/all/modules/hid/hid_profiles/hid_profiles.module

```diff
@@ -729,6 +729,14 @@ function hid_profiles_block_view($delta = '') {
 </div>  </div>';
       break;
     case 'hid_profiles_filters':
+      $node = node_load(arg(1));
```

Comment:
If we're using arg() we need to sanitize it a bit, for example is_numeric and if there's no node loaded break early.